.github/workflows/ci.yaml

```diff
@@ -302,8 +302,9 @@ jobs:
 
           if [ "${{ matrix.environment }}" = "production" ]
           then
-            yq w -d1 $DEPLOYMENT_NAME-without-host.yaml spec.values.service.additionalHosts[+] "scp.biomage.net" > $DEPLOYMENT_NAME.yaml
+            yq w -d1 $DEPLOYMENT_NAME-without-host.yaml spec.values.service.additionalHosts "true" > $DEPLOYMENT_NAME.yaml
           else
+            yq w -d1 $DEPLOYMENT_NAME-without-host.yaml spec.values.service.additionalHosts "false" > $DEPLOYMENT_NAME.yaml
             mv $DEPLOYMENT_NAME-without-host.yaml $DEPLOYMENT_NAME.yaml
```

Comment:
Won't this line replace the changes you made in the previous line?

Reply:
yes, it actually did and I removed it.